### DISCUSSION TITLE
Initialize new PocketBase instance with default/initial settings

### DISCUSSION THREAD

**Discussion by timharek** [pocketbase/pocketbase#5536] [Q&A] [answered]
I'm trying to initialize a new PocketBase instance with default/initial settings, but I keep getting an error:

```error
Error: Failed to fetch migrations list: SQL logic error: no such table: _collections (1); failed query: SELECT {{_collections}}.* FROM `_collections` ORDER BY `created` ASC
```

I have no `pb_data`-dir, but I have a `pb_migrations`-dir with the following file, `0_initial_settings.js`:

```js
/// <reference path="../pb_data/types.d.ts" />
migrate((db) => {
  const dao = new Dao(db);

  const settings = dao.findSettings();
  settings.meta.appName = "Hima";
  settings.logs.maxDays = 2;

  dao.saveSettings(settings);
});
```

Using PocketBase `v0.22.20`.

Shouldn't I be able to run my migration when I first do a `pocketbase serve`?

**Comment by ganigeorgiev** [comment, marked as answer]
Your migration timestamp is not "correct". Because all system and app migrations are stored together, the `0` index of your migration file will order it essentially first, aka. before the init system migration. Change the prefix to a valid unix timestamp (e.g. 1726503936).

**Reply by timharek** [reply]
Aha, I thought it would be valid since `0` would be the earliest/first, but not before the actual system init. I changed the timestamp to something closer to now, and it worked as expected!

Thanks for the quick reply, I really appreciate it!
